2022/project_idea.md

```diff
@@ -1,106 +1,116 @@
+To aid with Bayesian modeling analysis, ArviZ provides both numerical and visual summaries,
+as well as a collection of software tools that help with modeling and storing statistical results.
 
 ArviZ currently has a Julia and a Python interface and we are working on compatibility with R too.
 To date of writing, ArviZ integrates with at least 9 different Probabilistic Programming Languages (PPLs),
 including all major PPLs like Stan, PyMC or Turing.
 
-### Contact
-For any inquiry about Season of Docs at ArviZ, contact us at [gitter](https://gitter.im/arviz-devs/season_of_docs).
+### Contact 
+For any inquiry about Season of Docs at ArviZ, contact us at gitter.
 
 ## Project motivation
-Writing educational content to teach how to use ArviZ is hard, writing educational content to teach
-how to use PPLs is hard and writing content to teach how to use Bayesian modeling is also hard.
-However, the good news is that all these are not inherently hard because of some exceptional
-technical complexity. One huge challenge in writing such educational material is that
-all these tasks are actually one and the same, a wider task generally called
-[Bayesian workflow](https://arxiv.org/abs/2011.01808).
-
-Solving real problems with Bayesian modeling requires computational tools,
-so trying to explain Bayesian modeling without using any PPL is very limited.
-But at the same time, explaining how to use a PPL to someone who doesn't know
-anything about Bayesian modeling is also a fool's errand.
-And any modeling without visualization, diagnostics and model checking is
+Writing educational contents to teach how to use ArviZ  could pose a certain level of difficulty, producing educational contents to teach
+how to use PPLs and Bayesian modeling could equally pose a level of difficulty. This difficulty isn't because these frameworks and libraries are inherently complex or too technical but rather as a result of having an encompassing documentation that captures all
+the possible libraries that are used with the Bayesian workflow
+without doing a disservice to any group of users by not capturing their choice of library or language. 
+Here's the icebreaker, writing such educational material/guide is actually one and the same, 
+a wider task generally called Bayesian workflow.[Bayesian workflow](https://arxiv.org/abs/2011.01808].
+
+Solving real problems with the Bayesian model requires computational tools, 
+so trying to explain Bayesian modeling without using any PPL is very limited and not efficient. 
+But at the same time, explaining how to use a PPL to someone who doesn't know 
+anything about Bayesian modeling is also a fool's errand. 
+Any modeling without visualization, diagnostics and model checking is 
 doomed from the start.
-**We need to teach all these skills at the same time in an organized and coherent manner**
+**We need to teach all these skills at the same time in an organized and coherent manner**.
```

Comment:
```suggestion
**We need to teach all these skills at the same time in an organized and coherent manner**.

In the case of ArviZ in particular, we also aim to integrate with all available PPLs instead of only one.
Therefore, we need to write documentation that captures all
the possible libraries that are used with the Bayesian workflow.
Otherwise we would be doing a disservice to many of our users by not capturing their choice of library or language. 
The resource proposed here however is not limited to using ArviZ with multiple PPLs,
we also aim to include an R version for example (where ArviZ is not available) and
use alternatives to ArviZ.
```